### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by indigorose**
Added a navbar

### CODE REVIEW

**Comment by webtechalex** [comment]
I think the linting might not be working properly on this one. We have some extra spaces and some places where it has switched the quote style. I'll have a look into it. Maybe we can get this change merged so that we have the feature at least, then I (or anyone else) can have a look at the code style stuff

**Comment by indigorose** [comment]
That's fine, it might be clashes with the prettier files when configuring node last night. 

I forgot to mention yesterday that I also included some minor changes to the pages for responsivity on mobile/smaller screens.

File: src/components/Navbar.astro [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+						{item.label}
+					</a>
+				))
+			}
+		</div>
+	</div>
+</nav>
+
+<script is:inline>
+	const menuButton = document.getElementById('menu');
```
Comment (webtechalex) [inline src/components/Navbar.astro:97]: I wonder if there is a more idiomatic way to do this. Maybe it's just my React-centered brain that has made me a little allergic to the DOM API and this sort of thing is perfectly fine in astro 🤔 ...not sure. Any thoughts @cajogos ?

File: src/pages/coding-challenges.astro [hunk 2]
```diff
@@ -1,11 +1,12 @@
 ---
 import Layout from "../layouts/Layout.astro";
-import Logo from "../components/Logo.astro";
+// import Logo from "../components/Logo.astro";
```
Comment (webtechalex) [inline src/pages/coding-challenges.astro]: I think this one wants deleting 😁

File: src/pages/blog.astro [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+import Layout from '../layouts/Layout.astro'
+import Navbar from "../components/Navbar.astro"
+
+---
+
+<Layout title="Blog">
+<Navbar />
+
```
Comment (webtechalex) [inline src/pages/blog.astro]: Nitpick: redundant whitespace

File: src/pages/coding-challenges.astro (hunk 2 above)
Comment (indigorose) [reply src/pages/coding-challenges.astro]: Happy to remove this.

Comment (webtechalex) [reply src/pages/coding-challenges.astro]: Awesome. I'll watch for the change.
If you have time, can you also just give the eslint and prettier scripts a kick and commit and push any changes they make?
```
npm run lint
npm run format
```
Maybe we can clean up some of these extra spaces and make the PR smaller.

Comment (webtechalex) [reply src/pages/coding-challenges.astro]: No rush on these by the way! I know it's getting late! 😬 😁

**Comment by indigorose** [comment]
Updated the quotations and whitespace where I can. Noted that I made a change the FAQ cards, they weren't responsive on smaller screens but my solution has a glitch and reverted this back to the original for now. This could be another first issue for others.

**Comment by webtechalex** [comment]
We still have large amounts of unnecessary changes due to extra whitespace added in FAQCard.astro, and index.astro. ~@indigorose can you pull the spacing back in those files so that it matches the originals? I was hoping that if you ran the linting and formatting scripts that it would be done automatically, but maybe not? Once done and merged I will take a look at the eslint and prettier configs and try to figure out why the checks are passing when they should fail.~
Actually, scratch that. I'm going to merge it as it is and I can use the current state to debug the problem with the lint and format rules.

**Comment by webtechalex** [comment]
On the subject of the responsiveness changes: I definitely think it's a good idea to keep PRs focused on a single issue or task. Awesome that you want to GSD but no need to cram things and risk having to context-switch 😉

**Pull request merged**